cmake/SSGCommon.cmake

```diff
@@ -83,49 +83,19 @@ macro(ssg_build_bash_remediation_functions)
     )
 endmacro()
 
-macro(ssg_build_guide_xml PRODUCT)
-    if(SSG_SVG_IN_XCCDF_ENABLED)
-        add_custom_command(
-            OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/guide.xml"
-            COMMAND "${XSLTPROC_EXECUTABLE}" --output "${CMAKE_CURRENT_BINARY_DIR}/guide.xml" "${SSG_SHARED_TRANSFORMS}/includelogo.xslt" "${SSG_SHARED}/xccdf/shared_guide.xml"
-            MAIN_DEPENDENCY "${SSG_SHARED}/xccdf/shared_guide.xml"
-            DEPENDS "${SSG_SHARED_TRANSFORMS}/includelogo.xslt"
-            COMMENT "[${PRODUCT}-content] generating guide.xml (SVG logo enabled)"
-        )
-    else()
-        add_custom_command(
-            OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/guide.xml"
-            COMMAND "${CMAKE_COMMAND}" -E copy "${SSG_SHARED}/xccdf/shared_guide.xml" "${CMAKE_CURRENT_BINARY_DIR}/guide.xml"
-            MAIN_DEPENDENCY "${SSG_SHARED}/xccdf/shared_guide.xml"
-            COMMENT "[${PRODUCT}-content] generating guide.xml (SVG logo disabled)"
-        )
-    endif()
-
-    add_custom_target(
-        generate-internal-${PRODUCT}-guide.xml
-        DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/guide.xml"
-    )
-endmacro()
-
 macro(ssg_build_shorthand_xml PRODUCT)
-    file(GLOB OVERLAYS_DEPS "${CMAKE_CURRENT_SOURCE_DIR}/overlays/*.xml")
-    file(GLOB PROFILE_DEPS "${CMAKE_CURRENT_SOURCE_DIR}/profiles/*.xml")
-    file(GLOB_RECURSE XCCDF_RULE_DEPS "${CMAKE_CURRENT_SOURCE_DIR}/xccdf/*.xml")
-    file(GLOB_RECURSE SHARED_XCCDF_RULE_DEPS "${SSG_SHARED}/xccdf/*.xml")
+    execute_process(
+        COMMAND "${SSG_SHARED_UTILS}/yaml-to-shorthand.py" --product-yaml "${CMAKE_CURRENT_SOURCE_DIR}/product.yml" --bash_remediation_fns "${CMAKE_BINARY_DIR}/bash-remediation-functions.xml" --output "${CMAKE_CURRENT_BINARY_DIR}/shorthand.xml" list-inputs
+        OUTPUT_VARIABLE SHORTHAND_INPUTS_STR
+    )
+    string(REPLACE "\n" ";" SHORTHAND_INPUTS "${SHORTHAND_INPUTS_STR}")
```

Comment:
`SHORTHAND_INPUTS` is not used anywhere, command below should depend on it?

Reply:
Fixed